src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAMDesignator.cs

```diff
@@ -13,6 +14,204 @@ public void AMDesignator_GetInvariantInfo_ReturnsExpected()
+            yield return new object[] { new CultureInfo("hi-IN").DateTimeFormat, "am" };
+            yield return new object[] { new CultureInfo("hr-BA").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("hr-HR").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("hu-HU").DateTimeFormat, "de." };
+            yield return new object[] { new CultureInfo("id-ID").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("it-CH").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("it-IT").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("ja-JP").DateTimeFormat, "午前" };
+            yield return new object[] { new CultureInfo("kn-IN").DateTimeFormat, "ಪೂರ್ವಾಹ್ನ" };
+            yield return new object[] { new CultureInfo("ko-KR").DateTimeFormat, "오전" };
+            yield return new object[] { new CultureInfo("lt-LT").DateTimeFormat, "priešpiet" };
+            yield return new object[] { new CultureInfo("lv-LV").DateTimeFormat, "priekšpusdienā" };
+            yield return new object[] { new CultureInfo("ml-IN").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("mr-IN").DateTimeFormat, "AM" }; // म.पू.
+            yield return new object[] { new CultureInfo("ms-BN").DateTimeFormat, "PG" };
+            yield return new object[] { new CultureInfo("ms-MY").DateTimeFormat, "PG" };
+            yield return new object[] { new CultureInfo("ms-SG").DateTimeFormat, "PG" };
+            yield return new object[] { new CultureInfo("nb-NO").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("no").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("no-NO").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("nl-AW").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("nl-BE").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("nl-NL").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("pl-PL").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("pt-BR").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("pt-PT").DateTimeFormat, "da manhã" };
+            yield return new object[] { new CultureInfo("ro-RO").DateTimeFormat, "a.m." };
+            yield return new object[] { new CultureInfo("ru-RU").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sk-SK").DateTimeFormat, "AM" };
+            yield return new object[] { new CultureInfo("sl-SI").DateTimeFormat, "dop." };
+            yield return new object[] { new CultureInfo("sr-Cyrl-RS").DateTimeFormat, "AM" }; // пре подне
+            yield return new object[] { new CultureInfo("sr-Latn-RS").DateTimeFormat, "AM" }; // pre podne
```

Comment:
That originally in dotnet it was 'пре подне'. Do you think I should add a prefix: "ICU returns "...?

Reply:
Or some comment on the top of test cases explaining what is it, so you don't need to add the prefix in all places.
As I understood these are behavioural changes between ICU and hybrid mode for WASM, will we have these cases documented?

Reply:
Sounds good. I am planning one example per endpoint with the general info what Web API is used because 1) list of exceptions would be long 2) it can vary between hosts. Documentation will be pushed soon.